concourse/scripts/create-cf.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname $0)" && pwd)
+ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+FLY_TARGET=${FLY_TARGET:-$ATC_URL}
+
+env=${DEPLOY_ENV-$1}
+pipeline="deploy-cloudfoundry"
+config="${SCRIPT_DIR}/../pipelines/create-cf.yml"
+bosh_password=$("$SCRIPT_DIR"/s3curl.pl --id="$AWS_ACCESS_KEY_ID" --key="$AWS_SECRET_ACCESS_KEY" --  \
```

Comment:
We should discuss if this is the best way to manage this secret. 

We are not sure about two things: 

First, the idea of generating the password, store it in S3 and then forcing us to query it from S3 to define the next pipeline.  I wonder if it is better to create the password in the bosh pipeline definition in a similar way we do in https://github.com/alphagov/paas-cf/blob/master/vagrant/environment.sh#L17

Second, having to maintain a script in perl, which also adds a new dependency.

This topic is touched by a story in the backlog https://www.pivotaltracker.com/projects/1275640/stories/110059806